src/main/NWN/API/Object/NwCreature.cs

```diff
@@ -1864,6 +1864,30 @@ public void RemoveFeat(Feat feat)
       Creature.m_pStats.RemoveFeat((ushort)feat);
     }
 
+    /// <summary>
+    /// Restore all <see cref="NwCreature" /> spells per day for given level.
+    /// </summary>
+    public void RestoreSpells(byte level)
+    {
+      if (level < 0 || level > 9)
```

Comment:
```suggestion
      if (level > 9)
```
`level < 0` is a redundant check. `level` is unsigned, and cannot be less than 0.